helpful.el

```diff
@@ -1220,7 +1220,7 @@ buffer."
           ;; Look up this command in the keymap, its parent and the
           ;; global map. We need to include the global map to find
           ;; remapped commands.
-          (where-is-internal command-sym keymap nil t))
+          (where-is-internal command-sym (list keymap) nil t))
```

Comment:
See the comment above this line of code: we need to include the global map in this case, or we don't handle remappings of global commands. It's unfortunate there aren't tests for this.